src/core/apim.tf

```diff
@@ -293,7 +293,7 @@ resource "azurerm_api_management_api_version_set" "apim_user_group_ms" {
 
 module "apim_user_group_ms_v1" {
   source              = "git::https://github.com/pagopa/azurerm.git//api_management_api?ref=v2.12.5"
-  name                = format("%s-ms-user-group-api-v1", local.project)
+  name                = format("%s-ms-user-group-api", local.project)
```

Comment:
the `-v1` suffix was removed because it will be added automatically (otherwise it will end with `-v1-v1` suffix)